examples/src/main/python/ml/simple_params_example.py

```diff
@@ -36,18 +35,20 @@
     if len(sys.argv) > 1:
```

Comment:
@yanboliang @MLnick Do you mind If I ask your thoughts as well? I don't mind changing this example not to check the `sys.argv` or making another PR to check `sys.argv` in this way.